cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1431,17 +1762,116 @@ instance (Show addr, Show versionNumber, Show agreedOptions, ToObject addr,
                                        (Map.toList cmState)
           ]
 
-
-instance Show addr
+instance (Show addr, ToObject addr, ToJSON addr)
       => ToObject (ServerTrace addr) where
-  -- TODO: a better 'ToObject' instance
-  toObject _verb ev =
-    mkObject [ "kind" .= String "ServerTrace"
-             , "event" .= show ev ]
+  toObject _verb (TrAcceptConnection peerAddr)     =
+    mkObject [ "kind" .= String "AcceptConnection"
+             , "address" .= toObject _verb peerAddr
+             ]
+  toObject _verb (TrAcceptError exception)         =
+    mkObject [ "kind" .= String "AcceptErroor"
+             , "error" .= show exception
+             ]
+  toObject _verb (TrAcceptPolicyTrace policyTrace) =
+    mkObject [ "kind" .= String "AcceptPolicyTrace"
+             , "policy" .= toObject _verb policyTrace
+             ]
+  toObject _verb (TrServerStarted peerAddrs)       =
+    mkObject [ "kind" .= String "AcceptPolicyTrace"
+             , "addresses" .= toJSON (toObject _verb `map` peerAddrs)
+             ]
+  toObject _verb TrServerStopped                   =
+    mkObject [ "kind" .= String "ServerStopped"
+             ]
+  toObject _verb (TrServerError exception)         =
+    mkObject [ "kind" .= String "ServerError"
+             , "error" .= show exception
+             ]
+
+instance ToJSON MiniProtocolNum where
+  toJSON (MiniProtocolNum w) =
+    Aeson.object [ "kind" .= String "MiniProtocolNum"
+                 , "num" .= w
+                 ]
 
 instance Show addr
       => ToObject (InboundGovernorTrace addr) where
-  -- TODO: a better 'ToObject' instance
-  toObject _verb ev =
-    mkObject [ "kind" .= String "InboundGovernorTrace"
-             , "event" .= show ev ]
+  toObject _verb (TrNewConnection p caddr)            =
+    mkObject [ "kind" .= String "NewConnection"
+             , "provenance" .= show p
+             , "connectionId" .= toObject _verb caddr
```

Comment:
```suggestion
  toObject _verb (TrNewConnection p connId)            =
    mkObject [ "kind" .= String "NewConnection"
             , "provenance" .= show p
             , "connectionId" .= toObject _verb connId
```

Reply:
Let's use `connId :: ConnectionId peerAddr` for connection id (also in other places) rather than `caddr`.  Just for the sake of consistency.